app/konnect/dev-portal/index.md

```diff
@@ -40,8 +40,26 @@ For developers:
 
 * [Generate credentials for an application](/konnect/dev-portal/applications/dev-gen-creds/) - This guide walks you through the process of generating and deleting API keys on the Dev Portal platform.
 
+### Contextual developer analytics 
+
+You can view analytics for developer applications from within the {{site.konnect_short_name}} Dev Portal. This gives developers the ability to gain insights into all of their applications and the relationships between applications and APIs using a fully customizable dashboard. 
```

Comment:
Fully customizable dashboards is not correct. Background and some headers change based on configurations made in the dev portal appearance but you can't change the tiles that host the visualizations. Those tiles stay with white background, black text, etc.

Example:
![image](https://github.com/Kong/docs.konghq.com/assets/498114/8a8eb33a-dd55-4be8-bfa4-0317a3b55c59)

Page background is blue(ish).
Header is red(ish)
Tile background is still white.